src/plugins/gui/providers/bityProvider.ts

```diff
@@ -189,14 +189,20 @@ const fetchBityQuote = async (bodyData: BityQuoteRequest) => {
     body: JSON.stringify(bodyData)
   }
   const result = await fetch('https://exchange.api.bity.com/v2/orders/estimate', request)
-  if (result.ok) {
-    const newData = await result.json()
-    return newData
-  } else {
-    const bityErrorRes = asBityErrorResponse(await result.json())
-    if (bityErrorRes.errors.some((bityError: { code: string; message: string }) => bityError.code === 'amount_too_large')) {
-      throw new FiatProviderError({ providerId, errorType: 'overLimit' })
+  try {
+    if (result.ok) {
+      const newData = await result.json()
+      return newData
+    } else {
+      const bityErrorRes = asBityErrorResponse(await result.json())
+      if (bityErrorRes.errors.some((bityError: { code: string; message: string }) => bityError.code === 'amount_too_large')) {
+        throw new FiatProviderError({ providerId, errorType: 'overLimit' })
```

Comment:
This throw will get caught by the outer catch and re-packaged as a generic error. Only wrap line 197 with a try/catch that would run

`throw new Error('Bity: Unable to fetch quote: ' + (await result.text()))`